tests/smoke.rs

```diff
@@ -227,17 +227,30 @@ impl Project {
         Ok(())
     }
 
+    /// Calls `xargo build`
+    fn build_from_workdir(&self, target: &str, working_dir: &Path) -> Result<()> {
+        // Be less verbose
+        self.build_from_workdir_and_get_stderr(Some(target), working_dir)?;
+        Ok(())
+    }
+
     /// Calls `xargo build` and collects STDERR
     fn build_and_get_stderr(&self, target: Option<&str>) -> Result<String> {
+        self.build_from_workdir_and_get_stderr(target, &self.td.path())
+    }
+
+    fn build_from_workdir_and_get_stderr(&self, target: Option<&str>, working_dir: &Path) -> Result<String> {
         let mut cmd = xargo()?;
         cmd.arg("build");
 
+        cmd.args(&["--manifest-path", &self.td.path().join("Cargo.toml").to_str().unwrap()]);
```

Comment:
Doesn't this mean that now *all* tests run with `--manifest-path`?

Reply:
good catch! i'll fix this

Reply:
fixed

Reply:
fixed.